thyrel-web/src/components/home/ButtonModalJoin.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Button, Input, Modal } from 'rsuite';
+import BigButton from '../BigButton';
+
+type ModalJoinProps = {
+  identifier?: string;
+  onClick: (identifier: string) => void;
+};
+
+export default function ModalJoin({
+  identifier: urlIdentifier,
+  onClick,
+}: ModalJoinProps) {
+  const [open, setOpen] = React.useState(false);
+  const [identifier, setIdentifier] = React.useState('');
+  return (
+    <>
+      <BigButton
+        icon="angle-double-up"
+        size="lg"
+        onClick={() => {
+          if (urlIdentifier) onClick(urlIdentifier);
+          else setOpen(true);
+        }}>
```

Comment:
j'aime bien ce genre de syntaxe (c'est très perso)
```suggestion
        onClick={() => urlIdentifier ? onClick(urlIdentifier) : setOpen(true)}
```